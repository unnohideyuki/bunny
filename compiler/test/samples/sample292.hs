a = 42 :: Int
b = 5 :: Integer
c = 3.14 :: Float
d = pi :: Double

main = do print $ (realToFrac a :: Float)
          print $ (realToFrac b :: Float)
          print $ (realToFrac c :: Float)
          print $ (realToFrac d :: Float)
          print $ (realToFrac a :: Double)
          print $ (realToFrac b :: Double)
          print $ (realToFrac c :: Double)
          print $ (realToFrac d :: Double)

