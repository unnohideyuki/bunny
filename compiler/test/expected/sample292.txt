42.0
5.0
3.14
3.1415927
42.0
5.0
3.140000104904175
3.141592653589793
